minorminer/utils/parallel_embeddings.py

```diff
@@ -265,13 +411,20 @@ def find_sublattice_embeddings(
             for diversification of the embeddings found.
         seed: seed for the `numpy.random.Generator` controlling shuffling (if
             invoked).
+        timeout: total time allowed across all embeddings in seconds. Time elapsed is
+            checked before each sublattice search begins, and before each call of the
+            embedder function. The total runtime is thereby bounded by timeout plus the
+            time required by one call to embedder.
 
     Raises:
         ValueError: If the target graph ``T`` is not of type zephyr, pegasus, or chimera.
+            If the tile_embedding is incompatible with the source graph and tile.
 
     Returns:
         list: A list of disjoint embeddings.
     """
+
+    timeout = perf_counter() + timeout
```

Comment:
Changing input parameter meaning (relative -> absolute time) could be a source of bugs and confusion. Rather, call it something like:
```suggestion
    timeout_at = perf_counter() + timeout
```